### DISCUSSION TITLE
NFTs for The Stories

### DISCUSSION THREAD

**Discussion by LumenJack** [stellar/sorobanathon#138] [General]
Here we go!

Prologue:
**Duct Tape** (with sound)

https://litemint.com/items/GDRE2WJR3NOCCUAM5JPX5J7L4KN5IHN5APBKP2S63MFW3KARXSPASFJW/DuctTape

Chapter 1:
**Hello World** (with sound)
https://litemint.com/items/GB6JZZJZXFFWZ7TCVXNQS2SGC7AC3VLWLPZ5DYPAGWRFWAOORT234Q4U/HelloWorld

Chapter 2:
**Lava Sneks** (with sound)
https://litemint.com/items/GAZ6F6HIMHLVWIQUY4DADLSGGLI2OMQCC64NKIST4ATAF2TM65HHJHHG/LavaSneks

Chapter 3:
**Desert Mixology** (no sound)
https://litemint.com/items/GDXYO7HC2HIU6NCC7I5BNZB5ILGSGL5UPETYPQ4BFWIS7IWS35NTJC2H/Mixology

Chapter 4:
**Heel Kick** (no sound)
https://litemint.com/items/GCYTHGBZFDXGN7IGRFBAWCP7RHYXXHM2ZHLHAGR2KADJMZVZCBYQJH73/HeelKick

Chapter 5:
**Rotary Phone** (with sound)
https://litemint.com/items/GALUX2XK64G4EW43TPTV436OGS5K4HPXQ7V5BLPATCT6Y43W6LQ45HHI/RotaryPhone

Chapter 6:
**Blue Fish** (with sound)
https://litemint.com/items/GAN6DRW6IYZTID66HY7D4H65ILPBFK3TEPGB4BCFFSMPXSLJWD62AWEY/BlueFish

Or you could just check out the collection page:

https://litemint.com/explore?n=lumen.litemint.store&s=none&t=Stellar%20Quest%20Sorobanathon%20NFTs&c=soroban

Thanks it was a lot of fun!

**Comment by stellarjohn** [comment]
Hi [LumenJack](https://github.com/LumenJack)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Comment by LumenJack** [comment]
Thanks!
It was a lot of fun.  I spent close to 40 hours on those NFTs, that doesn’t mean they are good, but it was definitely not a low effort submission.  I enjoy doing the frame by frame animations in Procreate, it’s a fun process, but it takes some time.  Probably close to 10k individual layers in those 7 submissions.

I filled out the KYC, will that also work for the other Stellar Quest Bounties i haven’t claimed? 

Thanks again for doing this- it really got the artists and community excited.

Jack
